recipes-images/images/console-image-robotframework.bb

```diff
@@ -24,4 +24,17 @@ IMAGE_INSTALL:append = " \
     dfu-util \
 "
 
+
+inherit extrausers
+
+GROUPADD_PARAM:${PN} = " \
+    -g 100 users; \
+    -g 27 sudo; \
+"
+
+# jenkinsuser pwd: 'openssl passwd -6 pwd' generated passwd.
```

Comment:
Why do you use the insecure password 'pwd' and then hash it?

Reply:
i did not it is just an example, the right password is another